### PR TITLE
Feature/add compose

### PR DESCRIPTION
**Pull request by smuu**
Blocked by https://github.com/rollkit/gm/pull/1 and https://github.com/celestiaorg/celestia-node/pull/2069

Creating docker-compose files for testing the rollup against local-celestia-devnet and BSR.

Consider removing `init-local.sh` and `init-testnet.sh` scripts.

Closes https://github.com/celestiaorg/devops/issues/209

### CODE REVIEW

**Review by jcstein — APPROVED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ghcr.io/rollkit/gm:v0.1.0
+
+USER root
+
+# hadolint ignore=DL3018
+RUN apk --no-cache add \
+        curl \
+        jq
+
+COPY setup.sh entrypoint.sh /home/
```
[the remark this answers is not in the document]
Comment (smuu) [reply docker/Dockerfile:10]: Yeah, I was considering the same. But if someone wants to mount the home folder to persist data, the entrypoint would be overridden.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+> In order to use the rollup on a testnet, you must have funds in the testnet account.
+  On the first run of the celestia node, the node will generate a new account and print the address.
+  You can then use the address to request funds from celestia Discord server.
+
+To use the rollup on the testnet, follow these steps:
+
+1. Navigate to the directory containing the files.
+2. Create the key folder and assign the correct permissions to have a stable address:
+    - `mkdir -p ./data/celestia-keys`
+    - `sudo chown 10001:1001 ./data/celestia-keys`
```
Comment (jrmanes) [inline docker/README.md:45]: In the previous `Dockerfile`, we are not specifying the UID, is it ok to change the permissions here?

File: docker/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (jrmanes) [reply docker/Dockerfile:10]: makes sense 😉

**Review by jrmanes — APPROVED** [review]
just a comment, otherwise LGTM! 🚀

**Review by Bidon15 — APPROVED** [review]

File: docker/README.md (hunk 2 above)
Comment (smuu) [reply docker/README.md:45]: This Dockerfile is based on `rollkit/gm`, which creates the user. See [here](https://github.com/rollkit/gm/blob/main/Dockerfile#L20).

But there is a typo in the docs, thanks for finding it :D

**Pull request merged**